### PR TITLE
[16.0][MIG] edi_xml_oca: Migration to 16.0

### PR DESCRIPTION
**Pull request by QuocDuong1306**
Moved from `edi/oca` and migrated to 16.0

### CODE REVIEW

**Comment by simahawk** [comment]
the commit [DON'T MERGE] test-requirements.txt](https://github.com/OCA/edi-framework/pull/44/commits/8271b225589c8e79ed814ff59e9219489afa1c3b) seesm wrong. This is a normal test dep :wink:

**Review by simahawk — COMMENTED** [review]
LG but the test req commit

**Comment by QuocDuong1306** [comment]
I updated

File: edi_xml_oca/__manifest__.py [hunk 1]
```diff
@@ -7,12 +7,12 @@
...
-    "version": "14.0.1.1.0",
+    "version": "16.0.1.0.0",
     "development_status": "Alpha",
     "license": "LGPL-3",
     "author": "ACSONE,Camptocamp,Odoo Community Association (OCA)",
     "maintainers": ["simahawk"],
     "website": "https://github.com/OCA/edi-framework",
     "depends": ["edi_oca", "component"],
-    "external_dependencies": {"python": ["xmlschema"]},
+    "external_dependencies": {"python": ["xmlschema", "xmlunittest"]},
```
Comment (simahawk) [inline edi_xml_oca/__manifest__.py]: this should go only to tests-requirements

File: requirements.txt [hunk 2]
```diff
@@ -1,3 +1,4 @@
 # generated from manifests external_dependencies
 PyYAML
 xmlschema
+xmlunittest
```
Comment (simahawk) [inline requirements.txt]: not here, tests-req

**Review by simahawk — CHANGES_REQUESTED** [review]

**Review by simahawk — APPROVED** [review]

**Pull request merged**